Test.txt

```diff
@@ -1 +1,3 @@
-This is a test.
```

Comment:
Testing another reply.

---
In reply to: [58288395](https://github.com/kivancmuslu/CFTest/pull/1#discussion_r58288395) [](ancestors = 58288395)